home/home_2.1/TestCases.txt

```diff
@@ -0,0 +1,44 @@
+Test Case 1:
+Step 1: User starts a program; Expected Result: Program starts
+Step 2: User should enter a name "Alice"; Expected Result: Program should display "Hello, Alice"
+Step 3: Program creates a file "user_stats.txt"; Expected Result: Program should create a file "user_stats.txt"
```

Comment:
Program creates a file is not a step. Step should be an action performed by QA engineer or automation script. Therefore ER of step 3 should be a part of ER for Step 2.
Another thing is that in precondition you should specify the fact that it should be the first run of the program e.g. no file is present.